game-core/src/main/java/games/strategy/triplea/delegate/power/calculator/CombatValue.java

```diff
@@ -91,24 +94,49 @@ static CombatValue buildAaCombatValue(
         AvailableSupports.getSortedSupport(
             new SupportCalculator(
                 allEnemyUnitsAliveOrWaitingToDie, //
-                data.getUnitTypeList().getSupportAaRules(),
+                gameData.getUnitTypeList().getSupportAaRules(),
                 !defending,
                 false));
 
     return defending
         ? AaDefenseCombatValue.builder()
-            .gameData(data)
+            .gameData(gameData)
             .supportFromFriends(supportFromFriends)
             .supportFromEnemies(supportFromEnemies)
             .friendUnits(allFriendlyUnitsAliveOrWaitingToDie)
             .enemyUnits(allEnemyUnitsAliveOrWaitingToDie)
             .build()
         : AaOffenseCombatValue.builder()
-            .gameData(data)
+            .gameData(gameData)
             .supportFromFriends(supportFromFriends)
             .supportFromEnemies(supportFromEnemies)
             .friendUnits(allFriendlyUnitsAliveOrWaitingToDie)
             .enemyUnits(allEnemyUnitsAliveOrWaitingToDie)
             .build();
   }
+
+  static CombatValue buildNoSupportCombatValue(
```

Comment:
I added this because the `buildMainCombatValue` requires a territory and some places this is called doesn't have a territory.  And I didn't want to pass in null to `buildMainCombatValue` for territory.  With the bombard combat value change (#8097) removes the territory parameter so I could wait for that to merge and then redo this piece.